tests/TestCompose_of-of/docker-compose.yml

```
@@ -25,7 +25,7 @@ services:
     command: >
       /bin/bash -c "source /opt/openfoam4/etc/bashrc && 
       cd /home/precice/openfoam-adapter/tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam &&
-      sed -i 's|gather-scatter\"|gather-scatter\" exchange-directory=\"/home/precice/Data/Exchange/\" network=\"eth0\"|g' precice-config_serial.xml &&
+      sed -i 's|<m2n:sockets from="Fluid" to="Solid"/>|<m2n:sockets from="Fluid" to="Solid" exchange-directory=\"/home/precice/Data/Exchange/\" network=\"eth0\"/>|g' precice-config.xml &&
```

I found it, the insertion in the Solid-container fails:
```suggestion
      sed -i 's|<m2n:sockets from=\"Fluid\" to=\"Solid\"/>|<m2n:sockets from=\"Fluid\" to=\"Solid\" exchange-directory=\"/home/precice/Data/Exchange/\" network=\"eth0\"/>|g' precice-config.xml &&
```